docs/Samples/DurableIds/Pages/Index.cshtml.cs

```diff
@@ -64,11 +73,15 @@ private void LogWithDurableID()
             LogLaunch(_logger, _version, _instanceID);
         }
 
-        // StartSearchExample:LogLaunch
         [LoggerMessage(Level = LogLevel.Information, Message = "Launch, ver={version}, instantion={instantionID}")]
         static partial void LogLaunch(ILogger logger, string version, Guid instantionID);
-        // EndSearchExample:LogLaunch
         // EndExample:DurableId
+
+
+        // START_DEFINE : LogRandomGuid
+        [LoggerMessage(Level = LogLevel.Information, Message = "Loop, random ID={guid}")]
```

Comment:
commenting on this file, it seems that 
System.Diagnostics.Metrics, meterFactory are not needed